owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -70,6 +70,11 @@ class ReleaseNotesViewModel(
                 subtitle = R.string.release_notes_4_3_0_subtitle_accessibility_improvements,
                 type = ReleaseNoteType.ENHANCEMENT
             ),
+            ReleaseNote(
+                title = R.string.release_notes_4_4_1_title_file_size_0,
+                subtitle = R.string.release_notes_4_4_1_subtitle_file_size_0,
+                type = ReleaseNoteType.BUGFIX
+            ),
```

Comment:
Let's remove the other release notes and just leave this one 👍

Reply:
Done 👍🏻